pom.xml

```diff
@@ -84,7 +84,7 @@
 	</dependencies>
 
 	<properties>
-		<openMRSVersion>1.9.2</openMRSVersion>
+		<openMRSVersion>1.10.2</openMRSVersion>
```

Comment:
Those were the versions I understand was right. It is a bit contradictory to the requires statement but these changes followed a template given in TRUNK-4713.